test/integration/profiler_test.go

```diff
@@ -341,19 +379,26 @@ func TestCPUProfilerWorks(t *testing.T) {
 	defer noFramePointersCmd.Process.Kill()
 	dwarfUnwoundPid := noFramePointersCmd.Process.Pid
 
+	// Test unwinding JIT without frame pointers in the AoT code.
+	jitCmd := exec.Command("../../testdata/out/x86/basic-cpp-jit-no-fp")
+	err = jitCmd.Start()
+	require.NoError(t, err)
+	defer jitCmd.Process.Kill()
```

Comment:
I'd prefer `t.Cleanup` instead of `defer` in the tests. In this case it won't change anything but with nested tests behaviour would change.

Reply:
Sure thing